packages/client/src/generation/TSClient/Count.ts

```diff
@@ -3,25 +3,12 @@ import indent from 'indent-string'
 
 import type { DMMFHelper } from '../../runtime/dmmf'
 import { DMMF } from '../../runtime/dmmf-types'
-import {
-  getAggregateArgsName,
```

Comment:
Do you know if any of those functions can be removed as well?

Reply:
All used, mostly in `/TsClient/Model.ts`.